wgpu/tests/common/image.rs

```diff
@@ -71,80 +65,172 @@ pub fn calc_difference(lhs: u8, rhs: u8) -> u8 {
     (lhs as i16 - rhs as i16).unsigned_abs() as u8
 }
 
+fn add_alpha(input: &[u8]) -> Vec<u8> {
+    input
+        .chunks_exact(3)
+        .flat_map(|chunk| [chunk[0], chunk[1], chunk[2], 255])
+        .collect()
+}
+
+fn remove_alpha(input: &[u8]) -> Vec<u8> {
+    input
+        .chunks_exact(4)
+        .flat_map(|chunk| &chunk[0..3])
+        .copied()
+        .collect()
+}
+
+#[cfg(not(target_arch = "wasm32"))]
+fn print_flip(pool: &mut nv_flip::FlipPool) {
+    println!("\tMean: {:.6}", pool.mean());
+    println!("\tMin Value: {:.6}", pool.min_value());
+    for percentile in [25, 50, 75, 95, 99] {
+        println!(
+            "\t      {percentile}%: {:.6}",
+            pool.get_percentile(percentile as f32 / 100.0, true)
+        );
+    }
+    println!("\tMax Value: {:.6}", pool.max_value());
+}
+
+#[derive(Debug, Clone, Copy)]
+pub enum ComparisonType {
```

Comment:
Here (and/or in the nv-flip doc), it would be useful to explain what the values represent in more details, for example is the error metric a normalized value in the [0, 1] range ? If you know what a reasonable default might be for the mean error, it would be useful to document it here as well.

Reply:
Yeah I'll document - it's hard to describe what it is, it's basically just "from [0, 1] how different these images".